src/routes/JobsRoute.js

```diff
@@ -107,26 +117,37 @@ class JobsRoute extends React.Component {
     return get(this.props.resources, 'query', {});
   }
 
+  showToast(messageId, messageType = 'success', values = {}) {
```

Comment:
This is dangerous since you're using `this` in the function. Use the arrow syntax to ensure we don't get into a situation where `this` is undefined.